runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/DataflowBatchWorkerHarness.java

```diff
@@ -53,12 +54,14 @@ private DataflowBatchWorkerHarness(DataflowWorkerHarnessOptions pipelineOptions)
 
   /** Creates the worker harness and then runs it. */
   public static void main(String[] args) throws Exception {
+    BeamWorkerInitializers.runOnStartup();
```

Comment:
There is an assumption that this will be the first line ever for processing, right?

Is it possible to ensure this? If not, should we add a comment?

Reply:
Do you mean add a comment so that no one moves it in the future?

Reply:
Yes, that works.